alpaca/rest_test.go

```diff
@@ -171,6 +171,36 @@ func TestListOrders(t *testing.T) {
 	assert.Nil(t, orders)
 }
 
+func TestListOrdersV2(t *testing.T) {
```

Comment:
[minor] This is a pretty weak test. I understand the same is true for pretty much all the other test cases in this file, but it would be nice to improve it. A good test would cover at least two cases: one with an "empty" request, and one with all, or almost all possible options, both with assertions on the request. Please see marketdata/rest_test.go for examples.

Reply:
I changed the test to also check if the query params are set correctly.